types/stoppable/stoppable-tests.ts

```diff
@@ -1,12 +1,12 @@
-import * as http from 'http';
+import * as https from 'https';
 import stoppable = require('stoppable');
 
-const server: stoppable.StoppableServer = stoppable(http.createServer());
+const server: stoppable.StoppableServer = stoppable(https.createServer());
 server.stop();
 
-const server2: stoppable.StoppableServer = stoppable(http.createServer(), 10000);
+const server2: stoppable.StoppableServer = stoppable(https.createServer(), 10000);
 server2.stop();
 
-const server3: stoppable.StoppableServer = stoppable(http.createServer());
+const server3: stoppable.StoppableServer = stoppable(https.createServer());
```

Comment:
Can you have both `http` and `https` tested here to make sure it's not a breaking change?